core/client/src/main/java/alluxio/client/file/FileOutStream.java

```diff
@@ -141,8 +141,8 @@ public FileOutStream(AlluxioURI path, OutStreamOptions options, FileSystemContex
         } else {
           UnderFileSystem ufs = UnderFileSystem.Factory.get(mUfsPath);
           // TODO(jiri): Implement collection of temporary files left behind by dead clients.
```

Comment:
can you please add a comment here to explain why there's no need to call `prepareUfsPath` while `createParent` option is `false`?